terminator/install.bash

```diff
@@ -2,17 +2,17 @@
 
 if [ ! -d ~/.config/terminator ]
 then
-    tue-install-debug "creating ~/.config/terminator, because not existing yet"
-    mkdir -p ~/.config/terminator
+    tue-install-debug "creating $HOME/.config/terminator, because not existing yet"
+    mkdir -p "$HOME"/.config/terminator
 else
-    tue-install-debug "~/.config/terminator already exists"
+    tue-install-debug "$HOME/.config/terminator already exists"
 fi
 
 if [ -f ~/.config/terminator/config ]
 then
-    tue-install-debug "tue-install-cp config ~/.config/terminator/config2"
-    tue-install-cp config ~/.config/terminator/config2
+    tue-install-debug "tue-install-cp config $HOME/.config/terminator/config2"
+    tue-install-cp config "$HOME"/.config/terminator/config2
```

Comment:
Why use `"$HOME"? `~` is perfectly fine.

Its not used in the `if` statements and in any other script throughout the targets repo

Reply:
`~` doesn't resolve in quotes according to shellcheck

Reply:
Yeah but we don't require quotes anywhere here because the whole path is specified so `"$HOME"` can be replaced by `~`

Reply:
Changed it.